hadoop-ozone/dist/src/main/smoketest/compatibility/read.robot

```diff
@@ -155,3 +155,22 @@ HSync Lease Recover Can Be Used
     Pass Execution If    '${CLIENT_VERSION}' < '${HSYNC_VERSION}'    Client does not support HSYNC
     Pass Execution If    '${CLUSTER_VERSION}' < '${HSYNC_VERSION}'   Cluster does not support HSYNC
     Execute    ozone admin om lease recover --path=ofs://om/vol1/fso-bucket-${DATA_VERSION}/dir/subdir/file
+
+Key Info File Flag Should Be Set Correctly
+    Pass Execution If    '${CLUSTER_VERSION}' <= '${EC_VERSION}'   Cluster does not support 'file' flag
+    Pass Execution If    '${CLIENT_VERSION}' <= '${EC_VERSION}'    Client does not support 'file' flag
+
+    ${volume} =       Create Random Volume
+    ${bucket} =       Create bucket with layout    ${volume}   FILE_SYSTEM_OPTIMIZED
+
+    ${keypath} =      Set Variable    /${volume}/${bucket}/dir-${DATA_VERSION}/file-${DATA_VERSION}
+    ${dirpath} =      Set Variable    /${volume}/${bucket}/dir-${DATA_VERSION}/
+
+    Execute           ozone fs -mkdir -p ${dirpath}
+    Create Key        ${keypath}    ${TESTFILE}
```

Comment:
These are write operations, should be in `write.robot`.  Better yet, please use existing dir and key created here:

https://github.com/apache/ozone/blob/b5dbc08d06e4a9ebeedc827762e329e3593cc9d1/hadoop-ozone/dist/src/main/smoketest/compatibility/write.robot#L53-L57

https://github.com/apache/ozone/blob/b5dbc08d06e4a9ebeedc827762e329e3593cc9d1/hadoop-ozone/dist/src/main/smoketest/compatibility/write.robot#L41-L42